src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -1,699 +1,573 @@
+                          </div>
                         </div>
                       </div>
                     </div>
                   </div>
-                </div>
-              </div>
-              <div class="panel panel-primary">
-                <div class="panel-body">
-                  <div class="row">
-                    <div class="col-md-12">
-                      <label class="control-label">Send emails for</label>
-                    </div>
-                  </div>
-                  <div class="row">
-                    <div class="col-sm-2" data-toggle="tooltip" data-placement="top" title="If the student has not joined the course yet, an email containing the link to join the course will automatically be sent on session opening time.">
-                      <div class="checkbox">
-                        <label for="sendreminderemail_join">Join reminder
-                        </label>
-                        <input type="checkbox" id="sendreminderemail_join" disabled="">
-                      </div>
-                    </div>
-                    <div class="col-sm-3" data-toggle="tooltip" data-placement="top" title="Select this option to automatically send an email to students to notify them when the session is open for submission." disabled="">
-                      <div class="checkbox">
-                        <label>Session opening reminder
-                        </label>
-                        <input type="checkbox" name="sendreminderemail" id="sendreminderemail_open" value="FEEDBACK_OPENING">
-                      </div>
-                    </div>
-                    <div class="col-sm-3" data-toggle="tooltip" data-placement="top" title="Select this option to automatically send an email to students to remind them to submit 24 hours before the end of the session.">
-                      <div class="checkbox">
-                        <label for="sendreminderemail_closing">Session closing reminder</label>
-                        <input type="checkbox" name="sendreminderemail" id="sendreminderemail_closing" value="FEEDBACK_CLOSING">
-                      </div>
-                    </div>
-                    <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="Select this option to automatically send an email to students to notify them when the session results is published.">
-                      <div class="checkbox">
-                        <label for="sendreminderemail_published">Results published announcement</label>
-                        <input type="checkbox" name="sendreminderemail" id="sendreminderemail_published" value="FEEDBACK_PUBLISHED">
-                      </div>
+                  <div class="form-group">
+                    <div class="col-md-offset-5 col-md-3">
+                      <button class="btn btn-primary">Create Feedback Session</button>
```

Comment:
I think this is the button under the section "How do I create and schedule a new feedback session?".
Try clicking this button, I think the page will jump to the top of the section. I checked some other buttons, and saw that some of them have `onclick="return false"` to prevent anything from happening.